src/Typed/Resources/assets/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@symfony/ux-typed",
+    "description": "Typed integration for Symfony",
+    "license": "MIT",
+    "version": "1.0.0",
+    "symfony": {
+        "controllers": {
+            "typed": {
+                "main": "dist/controller.js",
+                "webpackMode": "eager",
+                "fetch": "eager",
+                "enabled": true
+            }
+        }
+    },
+    "peerDependencies": {
+        "typed.js": "^2.0",
+        "@hotwired/stimulus": "^3.0.0"
+    },
+    "devDependencies": {
+        "typed.js": "^2.0",
+        "@symfony/stimulus-testing": "^1.1.0",
```

Comment:
I believe you can remove this: it's included in the root `package.json`, and because we use workspaces, the tests will leverage it from there.

Reply:
Correct.